### PR TITLE
Add LFS control

### PR DESCRIPTION
**Pull request by dsm-kb**
Add ability to turn LFS on / off for repo

### CODE REVIEW

File: github/repos_lfs.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
```
Comment (gmlewis) [inline github/repos_lfs.go:1]: For new files, we give them the year that they are created.

```suggestion
// Copyright 2022 The go-github AUTHORS. All rights reserved.
```

File: github/repos_lfs_test.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
```
Comment (gmlewis) [inline github/repos_lfs_test.go:1]:
```suggestion
// Copyright 2022 The go-github AUTHORS. All rights reserved.
```

File: github/repos_lfs.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// EnableLfs turns the LFS feature ON for selected repo.
```
Comment (gmlewis) [inline github/repos_lfs.go:13]:
```suggestion
// EnableLFS turns the LFS (Large File Storage) feature ON for the selected repo.
```

File: github/repos_lfs.go [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+//
+// GitHub API docs: https://docs.github.com/en/rest/repos/lfs#enable-git-lfs-for-a-repository
+func (s *RepositoriesService) EnableLfs(ctx context.Context, owner string, repo string) (*Response, error) {
```
Comment (gmlewis) [inline github/repos_lfs.go:16]:
```suggestion
func (s *RepositoriesService) EnableLFS(ctx context.Context, owner, repo string) (*Response, error) {
```

File: github/repos_lfs.go [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+	resp, err := s.client.Do(ctx, req, nil)
+	if err != nil {
+		return resp, err
+	}
+
+	return resp, nil
+}
+
+// DisableLfs turns the LFS feature OFF for selected repo.
```
Comment (gmlewis) [inline github/repos_lfs.go:32]:
```suggestion
// DisableLFS turns the LFS (Large File Storage) feature OFF for the selected repo.
```

File: github/repos_lfs.go [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+//
+// Github API docs: https://docs.github.com/en/rest/repos/lfs#disable-git-lfs-for-a-repository
+func (s *RepositoriesService) DisableLfs(ctx context.Context, owner string, repo string) (*Response, error) {
```
Comment (gmlewis) [inline github/repos_lfs.go:35]:
```suggestion
func (s *RepositoriesService) DisableLFS(ctx context.Context, owner, repo string) (*Response, error) {
```

File: github/repos_lfs_test.go [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+		w.WriteHeader(http.StatusNoContent)
+	})
+
+	ctx := context.Background()
+	if _, err := client.Repositories.EnableLfs(ctx, "o", "r"); err != nil {
+		t.Errorf("Repositories.EnableLfs returned error: %v", err)
+	}
+
+	const methodName = "EnableLfs"
```
Comment (gmlewis) [inline github/repos_lfs_test.go:29]:
```suggestion
	const methodName = "EnableLFS"
```

File: github/repos_lfs_test.go [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+		w.WriteHeader(http.StatusNoContent)
+	})
+
+	ctx := context.Background()
+	if _, err := client.Repositories.DisableLfs(ctx, "o", "r"); err != nil {
+		t.Errorf("Repositories.DisableLfs returned error: %v", err)
+	}
+
+	const methodName = "DisableLfs"
```
Comment (gmlewis) [inline github/repos_lfs_test.go:55]:
```suggestion
	const methodName = "DisableLFS"
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @dsm-kb !
Just a few minor tweaks, please, then we should be ready for a second LGTM before merging.

File: github/repos_lfs.go [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+	resp, err := s.client.Do(ctx, req, nil)
+	if err != nil {
+		return resp, err
+	}
+
+	return resp, nil
+}
+
+// DisableLFS turns the LFS (Large File Storage) feature OFF for the selected repo
```
Comment (gmlewis) [inline github/repos_lfs.go:32]:
```suggestion
// DisableLFS turns the LFS (Large File Storage) feature OFF for the selected repo.
```

File: github/repos_lfs.go [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+//
+// Github API docs: https://docs.github.com/en/rest/repos/lfs#disable-git-lfs-for-a-repository
```
Comment (gmlewis) [inline github/repos_lfs.go:34]:
```suggestion
// GitHub API docs: https://docs.github.com/en/rest/repos/lfs#disable-git-lfs-for-a-repository
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Sorry I missed a couple minor nits.

**Review by gmlewis — APPROVED** [review]
Thank you, @dsm-kb !
LGTM.

Awaiting second LGTM from any other contributor to this repo before merging.

**Comment by dsm-kb** [comment]
Thank you @gmlewis for the LGTM :-)

**Review by raynigon — APPROVED** [review]
LGTM

@dsm-kb For ypur next PR it would be nice, if you could explain what you want to do in the PR description. Makes the review a bit easier :)

**Comment by gmlewis** [comment]
Thank you, @raynigon !
Merged.

**Pull request merged**